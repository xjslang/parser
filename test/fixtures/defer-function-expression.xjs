const testFunc = function() {
  console.log('function expression start')
  defer {
    console.log('function expression defer')
  }
  console.log('function expression end')
}

testFunc()
