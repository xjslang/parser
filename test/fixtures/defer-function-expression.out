function expression start
function expression end
function expression defer
